src/api/git/getChangedFiles.int.test.ts

```diff
@@ -78,10 +80,15 @@ it('should exclude file changes based on ignore parameter', async () => {
 
   await Promise.all([
     fs.promises.rm('a'),
-    fs.promises.writeFile('b', '2'),
-    fs.promises.writeFile('c', '2'),
+    fs.promises.writeFile('b', '21'),
```

Comment:
Not sure if bug but if the size of the contents remains the same it doesn't count as modified :\